signer/fulcio/fulcio_test.go

```diff
@@ -76,6 +78,37 @@ func TestNewClient(t *testing.T) {
 	require.NotNil(t, client)
 }
 
+func TestIDToken(t *testing.T) {
+	// test when supplying a raw token, the same token is returned
+	tok := generateTestToken("test@example.com", "testsubject")
+	out, err := idToken(tok)
+	require.NoError(t, err)
+	require.Equal(t, tok, out)
+
+	// test when supplying a path, a valid token is returned
+	// NOTE: this function could be refactored to accept a fileSystem or io.Reader so reading the file can be mocked,
+	// but unsure if this is the way we want to go for now
```

Comment:
I like this idea. The less code we have in the test (i.e. composing a file path) the better. I think you could also do a relative path: `../../hack/test.token`. I also don't have a strong preference on where to put test data like that.